packages/Webkul/BookingProduct/src/Resources/lang/ben/app.php

```diff
@@ -87,7 +87,7 @@
             'contact'                  => 'যোগাযোগ',
             'email'                    => 'ইমেইল',
             'slot-duration'            => 'স্লট সময়কাল',
-            'slot-duration-in-minutes' => ':minutes মিনিট',
+            'slot-duration-in-minutes' => 'স্লট-সময়কাল-মিনিটে',
```

Comment:
Hi,

You translated :minutes variable also, which will not provide the correct string.